src/processout/processout.ts

```diff
@@ -488,23 +513,22 @@ module ProcessOut {
             success: (token: string) => void,
             error:   (err: Exception) => void): void {
 
-            if (!this.projectID)
-                throw new Exception("default", "No project ID was set when instanciating ProcessOut.js. To refresh a card CVC, a project ID must be set.");
+            this.assertPKFetched(function() {
+                this.apiRequest("put", `cards/${cardUID}`, {
+                    "cvc": cvc
+                }, function(data: any, code: number, 
+                    req: XMLHttpRequest, e: Event): void {
 
-            this.apiRequest("put", `cards/${cardUID}`, {
-                "cvc": cvc
-            }, function(data: any, code: number, 
-                req: XMLHttpRequest, e: Event): void {
-
-                if (!data.success) {
-                    error(new Exception("card.invalid"));
-                    return
-                }
+                    if (!data.success) {
+                        error(new Exception("card.invalid"));
+                        return
+                    }
 
-                success(data.card.id);
-            }, function(code: number, req: XMLHttpRequest, e: Event): void {
-                error(new Exception("card.invalid"));
-            });
+                    success(data.card.id);
+                }, function(code: number, req: XMLHttpRequest, e: Event): void {
+                    error(new Exception("processout-js.network-issue"));
+                });
```

Comment:
This is fine but just for exemple purpose:

replacing the whole new function by error.bind(null, new Exception("procesout-js.network-issue")) works as well (but create the Exception before calling the parent function